psydac/linalg/basic.py

```diff
@@ -5,7 +5,21 @@
 from abc   import ABCMeta, abstractmethod
 from numpy import ndarray
 
-__all__ = ['VectorSpace', 'Vector', 'LinearOperator', 'LinearSolver', 'Matrix']
+__all__ = ['VectorSpace', 'Vector', 'LinearOperator', 'LinearSolver', 'Matrix', 'IdentityElement', 'NullElement']
+
+#===============================================================================
+class IdentityElement(metaclass=ABCMeta):
+    """
+    Dummy type for a general identity element.
+    """
+    pass
+
+#===============================================================================
+class NullElement(metaclass=ABCMeta):
+    """
+    Dummy type for a general null element.
+    """
+    pass
```

Comment:
The naming of these classes does not convince me, as I do not understand what "element" means here, neither in the name of the classes nor in the docstrings.

More importantly these classes are not used anywhere in Psydac, so they seem completely unnecessary to me. I recommend that we postpone their introduction to a future PR dedicated to the abstract linear operators.

Reply:
Ok, they're gone for now.